frontend/src/components/common/Table.tsx

```diff
@@ -0,0 +1,244 @@
+import { Paper } from '@mui/material';
```

Comment:
Can you please check if there is a performance regression or not? There's a few scripts in the load tests folder to generate some large tables which might be helpful.

Reply:
I tried it with load tests and bunch of objects and it works okay. There are no noticeable slowdowns compared to the previous table. One perf issue that is present is when table is quickly updated, but it's not caused by the table itself and is same with the old table. I looked a bit into it and adding some throttling to the useList functions in KubeObjects would probably improve perf in this case but it's out of scope for this pr

Reply:
Thanks for checking. It's good that there's no regression in performance.